extended/src/main/java/apoc/ml/bedrock/BedrockInvokeResult.java

```diff
@@ -0,0 +1,45 @@
+package apoc.ml.bedrock;
+
+import java.util.List;
+import java.util.Map;
+
+public class BedrockInvokeResult {
+    public record StabilityAi(String base64Image) {
+        public static StabilityAi from(Map<String, Object> map) {
+            String base64 = (String) map.get("base64");
+            
+            return new StabilityAi(base64);
+        }
+    }
+
+    public record AnthropicClaude(String completion, String stopReason) {
+        public static AnthropicClaude from(Map<String, Object> map) {
+            String completion = (String) map.get("completion");
+            String stopReason = (String) map.get("stop_reason");
+            
+            return new AnthropicClaude(completion, stopReason);
+        }
+    }
+
+    public record TitanEmbedding(Long inputTextTokenCount, List<Number> embedding) {
+        public static TitanEmbedding from(Map<String, Object> map) {
+            Long inputTextTokenCount = (Long) map.get("inputTextTokenCount");
+            List<Number> embedding = (List<Number>) map.get("embedding");
```

Comment:
probably use List<Double> instead, as Cypher doesn't really know how to deal with Number type.